MagicalCryptoWallet.Tests/NodeBuilding/CoreNode.cs

```diff
@@ -157,7 +160,7 @@ public Node CreateNodeClient(NodeConnectionParameters parameters)
 			File.WriteAllText(_config, config.ToString());
 			lock (_l)
 			{
-				_process = Process.Start(new FileInfo(_Builder.BitcoinD).FullName, "-conf=bitcoin.conf" + " -datadir=" + DataDir + " -debug=net");
+				_process = Process.Start(new FileInfo(_Builder.BitcoinD).FullName, "-conf=bitcoin.conf" + " -datadir=" + DataDir + " -debug=1");
```

Comment:
@lontivero I think it doesn't have an effect, does it? You must do printtoconsole 1, too in order to see the debug logs, like Nicolas is doing. However I turned that off, because that ruined the testing logs.

Reply:
`debug=1` is for logging for all loggers and not just network related log entries. The logs are a bit longer now but they have a very short live because they are deleted once the test finish. Let me know if you prefer I roll it back.

Reply:
It's good.